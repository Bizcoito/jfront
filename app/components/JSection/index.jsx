import React from 'react';
import PropTypes from 'prop-types';

const JSection = props => (
  <section className="section2">
    <div className="container2 is-fluid">
      <h1 className="title2">{props.title}</h1>
      <h2 className="subtitle2">{props.subtitle}</h2>
    </div>
  </section>
);

JSection.propTypes = {
  title: PropTypes.string.isRequired,
  subtitle: PropTypes.string.isRequired,
};

export default JSection;
