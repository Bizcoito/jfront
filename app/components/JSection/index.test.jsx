import React from 'react';
import renderer from 'react-test-renderer';
import JSection from './index';

test('JSection render title and subtitle props', () => {
  const component = renderer.create(
    <JSection title="JFront" subtitle="Javascript Single Page Application" />,
  );

  expect(component).toMatchSnapshot();
});
