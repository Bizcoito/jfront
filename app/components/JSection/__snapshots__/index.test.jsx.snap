// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JSection render title and subtitle props 1`] = `
<section
  className="section"
>
  <div
    className="container is-fluid"
  >
    <h1
      className="title"
    >
      JFront
    </h1>
    <h2
      className="subtitle"
    >
      Javascript Single Page Application
    </h2>
  </div>
</section>
`;
